src/types/person.rs

```diff
@@ -21,34 +22,59 @@ pub struct Person {
 }
 
 impl Person {
-    /// Constructs a new person from a chunk vector according to the specs of
-    /// [Nicolas Markey in "Tame the BeaST"][taming], pp. 23-24.
+    /// Constructs a new person from a chunk vector:
+    /// 1. according to the specs of
+    ///     [Nicolas Markey in "Tame the BeaST"][taming], pp. 23-24.
+    /// 2. biblatex extended name format according to the
+    ///     [documentation of biblatex][biblatex], section 3.4 pp. 80-81,
+    ///     and section §4.2.3 pp. 164-165.
+    ///     Support is limited to default `nameparts`: prefix, family,
+    ///     suffix, given.
     ///
     /// [taming]: https://ftp.rrze.uni-erlangen.de/ctan/info/bibtex/tamethebeast/ttb_en.pdf
+    /// [biblatex]: https://ctan.gutenberg-asso.fr/macros/latex/contrib/biblatex/doc/biblatex.pdf
     pub fn parse(chunks: ChunksRef) -> Self {
-        let num_commas: usize =
-            chunks
-                .iter()
-                .map(|val| {
-                    if let Chunk::Normal(s) = &val.v {
-                        s.matches(',').count()
-                    } else {
-                        0
-                    }
-                })
-                .sum();
+        // Extended Name Format
+        let num_equals = count_num_char(chunks, '=');
+        if num_equals > 0 {
+            return Self::parse_extended_name_format(chunks);
+        }
```

Comment:
This call could check for `contains` instead so we don't have to walk the whole string

Reply:
On that check, I'm not sure whether this would work in all cases, but it should work in most of them at least.